metricbeat/module/elasticsearch/ingest_pipeline/_meta/data.json

```diff
@@ -0,0 +1,58 @@
+{
+  "@timestamp": "2023-01-19T19:08:28.770Z",
+  "elasticsearch": {
+    "cluster": {
+      "id": "XXXX",
```

Comment:
Curious if we have issues publishing cluster uuids and node ids ? They are exposed in the other metricsets

Reply:
I didn't want to publish them from my personal cluster I was using to test :) I can copy the ones from the other examples though.